.github/ISSUE_TEMPLATE/feature-request-title.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request title
+about: Brief summary of changes
+title: FROS-[FEATURE]
+labels: enhancement
+assignees: ''
+
+---
+
+## The routes implemented are as follows:
+### `/api/v1/example/example1`
+#### GET
+Brief summary of route 
+
+### `/api/v1/example/example2`
+#### POST
+Brief summary of route 
+
+# Any additional remarks:
+Summary of additional changes
+
+# Testing:
+Describe how this feature should be tested
```

Comment:
# Feature Description
*Please describe the feature you would like added*

# Use Case
*Please describe the ideal use-case for this feature*

# Prerequisites
*Are there any prerequisites that need to be addressed prior to this feature?*

# Any additional remarks:
Summary of additional changes
```